src/Core/Managed/Net45/Extensibility/Implementation/External/UserContextData.cs

```diff
@@ -77,6 +77,15 @@ public string StoreRegion
         {
             get { return this.tags.GetTagValueOrNull(ContextTagKeys.Keys.UserStoreRegion); }
             set { this.tags.SetStringValueOrRemove(ContextTagKeys.Keys.UserStoreRegion, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the AuthUserId of an application-defined account associated with the user.
+        /// </summary>
+        public string AuthUserId

```

Comment:
This is a public class we expose to customers and we need to make sure that names are user friendly and well-explained. It doesn't matter what are the names of the actual schema fields